tests/ha/test_data_pod_failure.py

```diff
@@ -1190,152 +1148,143 @@ def test_writes_deletes_during_pod_down(self):
         s3_test_obj = S3TestLib(access_key=access_key, secret_key=secret_key,
                                 endpoint_url=S3_CFG["s3_url"])
 
-        LOGGER.info("Step 1: Performing WRITEs and DELETEs with variable object sizes. "
+        LOGGER.info("Step 1: Start WRITEs and DELETEs with variable object sizes "
                     "during data pod down by delete deployment.")
 
-        LOGGER.info("Starting WRITEs on %s buckets", wr_bucket)
+        LOGGER.info("Perform WRITEs on %s buckets for background DELETEs", del_total_bkt)
+        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
+                'skipget': True, 'skipdel': True, 'bkts_to_wr': del_total_bkt, 'output': wr_output}
+        self.ha_obj.put_get_delete(event, s3_test_obj, **args)
+        wr_resp = tuple()
+        while len(wr_resp) != 3:
+            wr_resp = wr_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
+        s3_data = wr_resp[0]  # Contains IAM user data for passed buckets
+        written_bck = s3_test_obj.bucket_list()[1]
+        assert_utils.assert_equal(len(written_bck), del_total_bkt,
+                                  f"Failed to create {del_total_bkt} number of buckets."
+                                  f"Created {len(written_bck)} number of buckets")
+        LOGGER.info("Performed WRITEs on %s buckets for background DELETEs", del_total_bkt)
+
+        LOGGER.info("Starting WRITEs on %s buckets in background", wr_bucket)
         args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
                 'skipget': True, 'skipdel': True, 'bkts_to_wr': wr_bucket, 'output': wr_output}
-
         thread1 = threading.Thread(target=self.ha_obj.put_get_delete,
                                    args=(event, s3_test_obj,), kwargs=args)
-        thread1.daemon = True  # Daemonize thread
-        thread1.start()
-        LOGGER.info("Successfully started WRITEs in background")
 
-        time.sleep(HA_CFG["common_params"]["20sec_delay"])
-        LOGGER.info("Starting DELETEs of %s buckets", del_bucket)
+        LOGGER.info("Starting DELETEs of %s buckets in background", del_bucket)
+        get_random_buck = self.system_random.sample(written_bck, del_bucket)
         args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
-                'skipput': True, 'skipget': True, 'bkts_to_del': del_bucket, 'output': del_output}
-
+                'skipput': True, 'skipget': True, 'bkt_list': get_random_buck,
+                'bkts_to_del': del_bucket, 'output': del_output}
         thread2 = threading.Thread(target=self.ha_obj.put_get_delete,
                                    args=(event, s3_test_obj,), kwargs=args)
```

Comment:
Please make sure two different users are being used for write and delete operations to avoid conflicts.

Reply:
> Please make sure two different users are being used for write and delete operations to avoid conflicts.

same as above
Sending specific list of bucket written for DELETEs only
get_random_buck = self.system_random.sample(written_bck, del_bucket)